### PR TITLE
Add eNotary component doc

### PR DESCRIPTION
**Pull request by SteveLasker**
Signed-off-by: Steve Lasker <stevenlasker@hotmail.com>

### CODE REVIEW

File: components.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title:  "Components"
+nav_order: 5
+has_children: true
+has_toc: true
+---
```
Comment (SteveLasker) [inline components.md:7]: This page is a stub to create the left hand navigation. The `has_toc: true` will show a table of contents, with a link to the page.
As we get more components, we can choose to make this page more robust than just the TOC entry.

**Review by knight-brian — APPROVED** [review]

File: components/enotary.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+---
+
+# eNotary
+
+SCITT provides a means to store information on a ledger. The information on the ledger is only as good as the validity of the identities for the content placed on the ledger.
+Similar to a public notary, an eNotary verifies the identity of the incoming requests.
+If the identity is verified, the eNotary will provide a notarized endorsement for the entry, issuing a [receipt](https://datatracker.ietf.org/doc/draft-birkholz-scitt-receipts/){:target="_blank"} for the completed process.
+
+The identify of the claims and/or evidence, is not tied to the identity of the user interacting with the SCITT APIs.
+For example, Owen is the user uploading the Wabbit Networks device driver, with the associated SBOM.
```
Comment (OR13) [inline components/enotary.md:15]: maybe introduce Owen with a job title, and description.

File: components/enotary.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+The device driver and SBOM are signed by Wabbit Networks.
+Owen is simply performing the operation.
+Wabbit Networks doesn't have access to the SCITT APIs to upload the driver, but Owen does.
+
+When Parker goes to install the Wabbit Networks device driver, they don't really care or know who Owen is, but the software must be signed by a trusted identity, in this case: Wabbit Networks.
```
Comment (OR13) [inline components/enotary.md:20]: Introduce Parker?

File: components/enotary.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+The trust is placed on the identity of the artifact, the access control limits are external to the SCITT functionality.
+
+The eNotary policy provides control over the identity provider types, such as x509 issued certificates, or additional DID Web based identities.
```
Comment (OR13) [inline components/enotary.md:23]:
```suggestion
The eNotary policy provides control over the identity provider types, such as x509 issued certificates, or additional identifiers.
```

File: components/enotary.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+Also similar to a public notary, the eNotary process does not attempt to validate the evidence being notarized.
+
+## eNotary Conceptual Overview
+
+The eNotary provides the following conceptual workflow:
+
+1. A claim about the Wabbit Networks device driver, (with optional evidence) is submitted to a SCITT instance
+2. On ingress, the SCITT eNotary parses the identity associated with the claim and compares the identity type with those supported in the eNotary policy. 
+3. If the eNotary policy supports the identity type, including x509 and DID providers, the eNotary will attempt to verify the validity of the identity.
```
Comment (OR13) [inline components/enotary.md:33]:
```suggestion
3. If the eNotary policy supports the identity type, including x509 and [DID](https://www.w3.org/TR/did-core/) providers, the eNotary will attempt to verify the validity of the identity.
```

**Review by OR13 — APPROVED** [review]

**Comment by SteveLasker** [comment]
@OR13, @knight-brian I've redone the workflow to incorporate the feedback and more clarity around claims and evidence.

**Pull request merged**